rust/connlib/connection/src/pool.rs

```diff
@@ -108,7 +115,22 @@ where
         };
 
         if let Some(agent) = self.agent_mut(id) {
-            agent.add_remote_candidate(candidate);
+            agent.add_remote_candidate(candidate.clone());
+        }
+
+        // Each remote candidate might be source of traffic: Bind a channel for each.
```

Comment:
Note to myself: Think about when we need channels. Like, do we ever need a channel that is _not_ to another relay candidate? Shouldn't we always be able to hole-punch otherwise?